### ISSUE TITLE
Team mfing blocking leader in a5

### ISSUE THREAD

**Issue by PoBErs** [kolton/d2bot-with-kolbot#808]
I have a party of three with 1 leader and 2 team mfer. Occasionally the 2 team mfer will block the stairs leading  from upper place, where the TP spawns down to the stash so the leader can't get past through.

Anyone else having this issue and could help me to edit the team mf players to not be in the way? Thank you!

**Comment by jaenster** [comment]
I need a bit more information with what your mean.

First of all i assume your talking about act 5, due to the mentioning of stairs that go up and down and the fact i know that is the only TP spot close to relatively long stairs.

Is this something that happens only in begin game (when they just enter the game), or if not, by which script this happens?

**Comment by PoBErs** [comment]
Yes, it is in A5.

It happens randomly when an area is cleared in A5 and the two team mfer is teleporting back to town waiting for the leader to go to the new area. Sometimes the two mf:er lines up fine and the leader can get through to the WP. But occassionaly they block the stairs.

It's not a matter of which scripts. Just A5.

What I do think the problem is, is within the team mfer scripts. I would like to move the team mfer to up for example up to the upper right corner to not block. 

This is also just for a 3player game. If it would been a 8p game it would have been a disaster in 5 because of all the blocking.

**Comment by DarkHorseDre** [comment]
afaik where the toons position themselves is common to all scripts - are you running trunk and svn updated?  I have not seen this issue in thousands of team runs (although not been in act5 this season as in classic atm)

**Comment by PoBErs** [comment]
yeah, trunk and svn updated. The rest of the acts are no problem due to big space but just that stairway in a5 bugs me out.

**Comment by DarkHorseDre** [comment]
thats weird man - check the limedrop update notes - maybe they fixed it there if not trunk?
